test/unit/org/apache/cassandra/service/reads/repair/AbstractReadRepairTest.java

```diff
@@ -294,7 +294,7 @@ public void setUp()
     static ReplicaPlan.ForRangeRead replicaPlan(Keyspace keyspace, ConsistencyLevel consistencyLevel, EndpointsForRange replicas, EndpointsForRange targets)
     {
         return new ReplicaPlan.ForRangeRead(keyspace, consistencyLevel,
-                ReplicaUtils.FULL_BOUNDS, replicas, targets);
+                ReplicaUtils.FULL_BOUNDS, replicas, targets, 1);
```

Comment:
Nit: I think we don't need to break this line

Reply:
+1